src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -909,6 +909,32 @@ export class BaseEditCourseForm extends React.Component {
               className="course-skill"
             />
             )}
+            <Field
+              name="organization_short_code_override"
+              component={RenderInputTextField}
+              label={<FieldLabel text="Organization Short Code Override" optional />}
+              extraInput={{ onInvalid: this.openCollapsible }}
+              disabled={disabled}
+              optional
+            />
+            <Field
+              name="organization_logo_override_url"
+              component={ImageUpload}
+              label={(
+                <FieldLabel
+                  id="organization_logo_override.label"
+                  text="Organization Logo Override"
+                />
+              )}
+              extraInput={{ onInvalid: this.openCollapsible }}
+              id="organization-logo-override"
+              className="course-image"
+              maxImageSizeKilo={256}
+              requiredWidth={110}
+              requiredHeight={110}
```

Comment:
To do internal validation/confirmation on this.

Reply:
Added comment on ticket and PR description